### ISSUE TITLE
Option to show either notification preview or notification badge

### ISSUE THREAD

**Issue by Rexx-DZN** [8bitPit/Niagara-Issues#2582]
### Is your feature request related to a problem?

To keep the launcher useable and clean there should be an added feature that adds notification badges instead of notification preview texts. Since many people actually have their Niagara homescreen setup with only the icons. Now the text previews ruin the experience most of the time and if u disable that u would lose a core function. Which is knowing if u have notifications, this time from badges. 

### Describe the solution you'd like

Basically either have a switch from either using text previews or notification badges to keep it clean.

### Feature Request Procedure

- [X] I have read the [knowledge base article on feature requests](https://help.niagaralauncher.app/article/44-feature-requests) and followed the steps mentioned

**Comment by justme30s** [comment]
.

**Comment by CerberusCode** [comment]
This could be a nice idea

**Comment by mrorms** [comment]
This will be much much better and waaay more cleaner than what we have now

**Comment by NiagaraSupport** [comment]
Hey there, and thanks for the request.
Implementing this is outside our immediate agenda, but we see the appeal. We'll continue collecting feedback, and we'll consider implementing it when the time is right.

**Comment by stenbod** [comment]
I would really like this. The screen is cluttered with text when you have pending notifications. Sometimes you want to keep the notifications to remember stuff. Very stressful to see all the text, and the only way to get rid of it is to swipe the notification away (an possibly forget something) or turn the notifications off. A badge with the number of notifications and a swipe to see the notification would be enough. Would really make the screen cleaner.

**Comment by RalfEs73** [comment]
Yes! Please add that feature! 🤟
